doc/changelog.md

```diff
@@ -9,30 +9,50 @@
   [(#650)](https://github.com/PennyLaneAI/catalyst/pull/650)
   [(#649)](https://github.com/PennyLaneAI/catalyst/pull/649)
   [(#661)](https://github.com/PennyLaneAI/catalyst/pull/661)
+  [(#621)](https://github.com/PennyLaneAI/catalyst/pull/621)
 
   Catalyst now supports callbacks with parameters and return values.
   The following is now possible:
 
   ```py
-  @pure_callback
+  @debug.callback
   def foo(val):
     return val
```

Comment:
This doesnt showcase the "debug" aspect very well, maybe an f-string print would be nice?

Reply:
Ah yes I did `git push -f` after the update and didn't update what I had done in the online code editor.